src/lib.rs

```diff
@@ -281,9 +281,6 @@ impl From<Keysym> for u32 {
     }
 }
 
-/// The "empty" keyboard symbol.
-pub const NO_SYMBOL: Keysym = Keysym(0);
```

Comment:
Would it not be confusing to have both? One crate could export NO_SYMBOL and the other crate Keysym::NoSymbol. It would look like they are not equal, even though they are

Reply:
I re-added it